package/kedro_viz/server.py

```diff
@@ -215,7 +215,7 @@ def _call_viz(host=None, port=None, browser=None, load_file=None, save_file=None
                 sys.exit(1)
     else:
         pipeline = get_project_context("create_pipeline")()
-        catalog = get_project_context("create_catalog")()
+        catalog = None
```

Comment:
sorry I should have been clearer, I meant `get_project_context("create_catalog")(None)`

Reply:
This causes test failures, any suggestions?